modules/Microsoft.Sql/managedInstances/.test/min/dependencies.bicep

```diff
@@ -0,0 +1,289 @@
+@description('Required. The name of the Virtual Network to create.')
+param virtualNetworkName string
+
+@description('Required. The name of the Network Security Group to create.')
+param networkSecurityGroupName string
+
+@description('Required. The name of the Route Table to create.')
+param routeTableName string
+
+@description('Optional. The location to deploy resources to.')
+param location string = resourceGroup().location
+
+var sqlMiVnetAddressPrefix = '10.0.0.0/16'
```

Comment:
Same as the other comment. If those variables are used only in a single location, I'd much rather put them there like all the other values

Reply:
The subnet prefix is widely used. The vnet prefix is related to the subnet one. If for any reason one would need to change their value, they would likely both be updated, so I suggest to keep them close.

Reply:
Hm OK. Personally, I feel it is confusing to introduce values in other places if not absolutely necessary, but I won't block the PR for it.